package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "grunt-fh-build",
   "description": "A plugin for development and build lifecycle of FeedHenry components",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
I think this should be a major version bump. Consuming modules will need to change things to get the same behaviour as before:

* Add a .npmignore or whatever to match what's expected
* Add a bundledDependencies field to package.json to get the same behaviour from `grunt fh-dist --only-bundle-deps`